arcade/camera/camera_2d.py

```diff
@@ -165,26 +101,75 @@ def from_raw_data(
             )
 
         _pos = position or (half_width, half_height)
-        _data = CameraData(
+        self._camera_data = CameraData(
             position=(_pos[0], _pos[1], 0.0),
             up=(up[0], up[1], 0.0),
             forward=(0.0, 0.0, -1.0),
             zoom=zoom
         )
-
-        _projection: OrthographicProjectionData = OrthographicProjectionData(
+        self._projection_data: OrthographicProjectionData = OrthographicProjectionData(
             left=left, right=right,
             top=top, bottom=bottom,
             near=near, far=far,
             viewport=viewport or (0, 0, width, height)
         )
+        self._ortho_projector: OrthographicProjector = OrthographicProjector(
+            window=self._window,
+            view=self._camera_data,
+            projection=self._projection_data
+        )
+
+    @classmethod
+    def from_camera_data(cls, *,
+                         camera_data: Optional[CameraData] = None,
+                         projection_data: Optional[OrthographicProjectionData] = None,
+                         render_target: Optional[Framebuffer] = None,
+                         window: Optional["Window"] = None) -> Self:
+        """
+        Create a Camera2D with reusing provided CameraData or/and OrthographicProjectionData object.
```

Comment:
How about:
```
Create a Camera2D from provided CameraData or/and OrthographicProjectionData objects.
```

Because it's possible that someone may use this function without another Camera. They may just create their own `CameraData` and/or `OrthographicProjectionData` objects.

Reply:
Ok